grafana-plugin/integration-tests/utils/constants.ts

```diff
@@ -1,8 +1,4 @@
-export const BASE_URL = process.env.BASE_URL || 'http://localhost:3000';
```

Comment:
this is instead now stored in the `playwright.config.ts` ([here](https://github.com/grafana/oncall/pull/1470/files#diff-ddb6dcbef7df07a85ba657ae71acf9551744a157aba836c3e821630140e6b76dR42)) and natively used by `page.goTo` calls